package.json

```diff
@@ -37,65 +37,40 @@
     "dist/**/package.json",
     "src/**/*.ts"
   ],
-  "moduleTypes": {
-    "jest.config.js": "cjs"
-  },
   "scripts": {
     "lint": "eslint src/ --ext .js,.jsx,.ts,.tsx",
-    "test": "jest --verbose",
+    "test": "mocha",
     "clean": "rm -rf dist build package",
     "ts-node": "ts-node",
     "docs": "typedoc --entryPoints src/main.ts",
     "docs:html": "typedoc --entryPoints src/main.ts --plugin none",
     "build:esm": "tsc -p tsconfig.json",
     "build:cjs": "tsc -p tsconfig.cjs.json",
-    "build:esbuild": "esbuild src/main.ts --bundle --minify --sourcemap=external --outfile=dist/esbuild/main.js --format=esm  && node ./node_modules/add-js-extension/dist/bin.js ./dist/esm --once",
+    "build:esbuild": "esbuild src/main.ts --bundle --minify --external:node:* --sourcemap=external --outfile=dist/esbuild/main.js --format=esm  && node ./node_modules/add-js-extension/dist/bin.js ./dist/esm --once",
     "build:dev": "esbuild src/main.ts --bundle --outfile=dist/esbuild/main.js --format=esm --sourcemap=inline",
     "build:watch": "esbuild src/main.ts --bundle --watch --outfile=dist/esbuild/main.js --format=esm && node ./node_modules/add-js-extension/dist/bin.js ./dist/esm --once",
-    "build:all": "npm run build:esm && npm run build:cjs && npm run build:esbuild && ./fixup.sh"
+    "build:all": "npm run build:esm && npm run build:cjs && npm run build:esbuild && ./fixup.sh",
+    "prepublishOnly": "build:all"
```

Comment:
I think this will need to be `"prepublishOnly": "npm run build:all".  You can see if it works by doing `npm publish --dry-run`